### PR TITLE
Fixes all problems detected by SpacemanDMM

### PR DESCRIPTION
**Pull request by Exxion**
Most of these changes don't do anything except make SpacemanDMM not complain. A couple types had to be renamed because the SpacemanDMM parser can't handle paths containing `operator`. Now that those paths have been renamed, we don't need to tell OpenDream to allow them, so I also removed that. I left `__opendream.dm` in because it might be useful for some other reason later.

I can't imagine how any of these changes could break anything, but I should test them anyway. I'll do that later.

### CODE REVIEW

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -1028,7 +1028,7 @@
 	if(M.bodytemperature > 310) //copypasted from the cold drinks check so I don't have to change minttox internally and maybe most certainly break shit in the process
 		M.bodytemperature = max(310, M.bodytemperature + (-5 * TEMPERATURE_DAMAGE_COEFFICIENT)) //that minty freshness my dude, chill out
 
-	if(fatgokaboom && M_FAT in M.mutations)
+	if(fatgokaboom && (M_FAT in M.mutations))
```
Comment (Exxion) [inline code/modules/reagents/Chemistry-Reagents.dm:1031]: This is technically logically different, but I don't think there is any reachable condition in which it actually changes the result.

**Comment by ValkyrieSkies** [comment]
This'll be a pretty big win if it fixes StrongDMM.

**Comment by ValkyrieSkies** [comment]
> This'll be a pretty big win if it fixes StrongDMM.

image
Very happy to report that it does in fact fix it.

**Review by Kurfursten — APPROVED** [review]
After you are satisfied with testing you can self-merge, code looks fine

**Comment by Exxion** [comment]
Everything appears to work. Self-merging as Kurf suggested.

**Pull request merged**